tests/unit/testUtils.ts

```diff
@@ -1,9 +1,17 @@
 import * as vscode from "vscode";
 import { KafkaTopicOperation } from "../../src/authz/types";
+import { ResponseError as DockerResponseError } from "../../src/clients/docker";
+import { ResponseError as FlinkArtifactsResponseError } from "../../src/clients/flinkArtifacts";
+import { ResponseError as FlinkComputePoolResponseError } from "../../src/clients/flinkComputePool";
+import { ResponseError as FlinkSqlResponseError } from "../../src/clients/flinkSql";
+import { ResponseError as KafkaResponseError } from "../../src/clients/kafkaRest";
 import { TopicData, TopicDataFromJSON } from "../../src/clients/kafkaRest/models";
-import { ResponseError } from "../../src/clients/sidecar";
+import { ResponseError as ScaffoldingServiceResponseError } from "../../src/clients/scaffoldingService";
+import { ResponseError as SchemaRegistryResponseError } from "../../src/clients/schemaRegistryRest";
+import { ResponseError as SidecarResponseError } from "../../src/clients/sidecar";
 import { EXTENSION_ID } from "../../src/constants";
 import { setExtensionContext } from "../../src/context/extension";
+import { AnyResponseError } from "../../src/errors";
```

Comment:
I'd like to move this (and probably the new `ResponseErrorSource`) up into `src/clients/`, but that would likely affect too many other additional files and this PR is already bigger than originally intended